Update entity_types.md (#30096)
DataDog/documentation: content/en/internal_developer_portal/software_catalog/entity_model/entity_types.md

You can define custom entity types beyond service, system, datastore, queue, and API. Custom entities allow you to represent any component or resource that is important to your organization but does not fit into the standard categories.

> First, define the kinds you want to use with [this API][3]. Only entities of the kinds you've explicitly setup will be accepted. In the example below, a user is declaring a library with links, tags, and owning teams.

Reviewer: ```suggestion
First, define the kinds you want to use with [this API][3]. Only entities of the kinds you've explicitly setup are accepted. In the example below, a user is declaring a library with links, tags, and owning teams.
```
